src/crm/dynamicsWebApi.ts

```diff
@@ -70,9 +73,10 @@ const getTestStationEntities = async (requestUrl: string): Promise<DynamoTestSta
   return lastValueFrom(
     axios.get<ApiFormat>(requestUrl).pipe(
       map((data) => data.data),
-      map((data: ApiFormat) => data.value.map((obj) => createDynamoTestStation(obj))),
+      map((data: ApiFormat) => data.value.filter((obj) => siteList.includes(obj.dvsa_premisecodes))),
```

Comment:
Can we add the `siteList` to the filter so we're only querying the records we're interested in, rather than getting everything that might have changed and then filtering?

Reply:
I had a look into this using the `$filter=[Property] in [SiteList]` but it didn't seem like CE CRM had it available
![image](https://user-images.githubusercontent.com/92087051/151797654-7ae94a74-00f7-4d79-b354-6ec69c9c7ecd.png)

Reply:
Ultimately we'll be removing this restriction anyway so I'm happy we tried! 😃